### PR TITLE
[Website] Update footer with adjunct legal links

### PR DESCRIPTION
**Pull request by DragosRistici**
### Related Ticket(s)

{{Provide url(s) to the related ticket(s) that this pull request addresses}}

### Description

{{Add a human-readable description / detail summary of what the PR is changing and any details around how and why}}

{{If applicable, include a screenshot indicating an example or examples of what the PR is changing in the application}}

### Changelog

**New**

- {{new thing}}

**Changed**

- {{changed thing}}

**Removed**

- {{removed thing}}

### CODE REVIEW

File: src/pages/components/footer.mdx [hunk 1]
```diff
@@ -11,62 +11,243 @@ import ComponentFeedback from "components/ComponentFeedback";
...
+
+![Image of micro footer](../../images/component/footer/footer_default_02.jpg)
+
+<Caption>Short Footer at xlg 1312px breakpoint, with one optional adjunct legal link</Caption>
 
 <br />
 
+#### When to use
+Use up to two adjunct legal links for legally required notes or messages legally required to provide, regarding changes, policies or trademark issues.
+<br>
```
Comment (kennylam) [inline src/pages/components/footer.mdx:93]: @DragosRistici The deploy preview is failing because `<br>` must be closed:

```suggestion
<br />
```

**Review by kennylam — CHANGES_REQUESTED** [review]
Thanks @DragosRistici, only thing is to close all `br` tags so the deploy preview can build – should be `<br />`

File: src/pages/components/footer.mdx (hunk 1 above)
Comment (DragosRistici) [reply src/pages/components/footer.mdx]: thx

File: src/pages/components/footer.mdx [hunk 2]
```diff
@@ -11,64 +11,262 @@ import ComponentFeedback from "components/ComponentFeedback";
...
+
+
+
+
+
+### Micro
+
+![Image of minimal footer](../../images/component/footer/footer_micro_anatomy.jpg)
+
+<Caption>Micro footer at max 1584px breakpoint/Caption>
```
Comment (kennylam) [inline src/pages/components/footer.mdx:153]: @DragosRistici missing a bracket here

```suggestion
<Caption>Micro footer at max 1584px breakpoint</Caption>
```

**Pull request merged**